bio/mehari/download-clinvar-db/wrapper.py

```diff
@@ -0,0 +1,39 @@
+__author__ = "Till Hartmann"
+__copyright__ = "Copyright 2025, Till Hartmann"
+__email__ = "till.hartmann@bih-charite.de"
+__license__ = "MIT"
+
+from snakemake.shell import shell
+
+import re
+
+version_re = re.compile(r"(?P<date>\d{8})\+(?P<annonars>\d+\.\d+\.\d+)")
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+version = snakemake.params.get("version", "")
+if not version:
+    raise ValueError("Missing required parameter: version")
+match = version_re.fullmatch(version)
```

Comment:
Again, if the param is mandatory, just to simply the code:

```suggestion
match = version_re.fullmatch(snakemake.params.version)
```

Reply:
Here https://github.com/snakemake/snakemake-wrappers/pull/4274#discussion_r2147031471 you suggested otherwise, but I agree with just accessing it this way as version is indeed a required parameter.